### ISSUE TITLE
Bug: Ferdium icon is missing from tray in Gnome 43.1

### ISSUE THREAD

**Issue by roiz-git** [ferdium/ferdium-app#908]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.4

### What Operating System are you using?

Other Linux

### Operating System Version

ArchLinux

### What arch are you using?

x64

### Last Known Working Ferdium version

6.2.3

### Expected Behavior

System tray  icon is shown always

### Actual Behavior

System tray icon is missing

### Steps to reproduce

upgrade from 6.2.3 to 6.2.4

**Comment by toalex77** [comment]
Same here: openSUSE Tumbleweed, KDE 5.26.5

**Comment by romario74** [comment]
Same here, Manjaro Cinnamon

**Comment by ftpd** [comment]
Same here, qtile on Gentoo. Version 6.2.3 works great.

**Comment by ArvinZaker** [comment]
same here, KDE plasma 5.26.5 endavourOS.

**Comment by tiredofit** [comment]
Issue occurs Arch Linux i3 v4.22
Built from latest git 7fc4ed9f and shows up (6.2.5)

**Comment by DewitteRuben** [comment]
Same here, Fedora 37 (Gnome 43.2)

**Comment by kerbmkii** [comment]
Fixed for me on 6.2.6 / Gnome 43.3

**Comment by ftpd** [comment]
6.2.6 didn't fix it for me (qtlie on Wayland, both appimage and flatpak version).

**Comment by roiz-git** [comment]
Fixed for me 6.2.6 Gnome 44
now I wonder who should close the issue?

**Comment by s7sost** [comment]
It's actually not working for me, I'm on Ubuntu 22.04, Ferdium version 6.2.6. It seemed to work with the previous version, this issue started when I updated it.

**Comment by marie-bnl** [comment]
Did you make sure this option is enabled?

![image](https://user-images.githubusercontent.com/39555268/235319770-4f6f3947-d163-4ca8-a519-5d9802d81a92.png)

**Comment by s7sost** [comment]
Perhaps I've should've mentioned it in the reply, but yes, it was already active. Since then, I've switched it on and off, restarted it with the switch on and then turned it off and on again to see it if appeared, but nothing. 
![Screenshot from 2023-04-29 15-40-54](https://user-images.githubusercontent.com/7074667/235321298-995303b5-5a00-40d5-a2c1-6706e8c301a4.png)

Edit: For further reference, I installed Ferdium using the deb amd64 version.

**Comment by marie-bnl** [comment]
And I guess that other applications with a tray icon do show up in your tray?

**Comment by s7sost** [comment]
Indeed: 
![image](https://user-images.githubusercontent.com/7074667/235324929-90f96307-8056-401a-87e7-da88780e5154.png)
I've already restarted and everything, so I have no clue what happened. None of those tray elements are new either, they were installed before Ferdium, worked in the previous Ferdium version, and it just stopped showing up after the update.

**Comment by marie-bnl** [comment]
Are there any logs if you launch Ferdium from the command line?

**Comment by s7sost** [comment]
This is what I get when I run it through the cmd:
```
$ ferdium
MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete
[MobX] Since strict-mode is enabled, changing (observed) observable values without using an action is not allowed. Tried to modify: Settings@10.store
Ferdium Migration:  🧙‍  Running database migration wizard
info: serving app on http://localhost:46569
Ferdium Migration:  🔧  Nothing to migrate, already on the newest version
[87829:0429/182401.058182:ERROR:interface_endpoint_client.cc(695)] Message 6 rejected by interface blink.mojom.WidgetHost
[87829:0429/182401.058339:ERROR:interface_endpoint_client.cc(695)] Message 7 rejected by interface blink.mojom.WidgetHost
[87829:0429/182401.058476:ERROR:interface_endpoint_client.cc(695)] Message 3 rejected by interface blink.mojom.Widget
APPIMAGE env is not defined, current application is not an AppImage
```

**Comment by mcmxcdev** [comment]
Is this still an issue for anyone in this thread? Can't reproduce this with either the AppImage or Flatpak version of Ferdium.

**Comment by ftpd** [comment]
Works fine now. Some Electron update on they way had to fix it.

**Comment by s7sost** [comment]
> Is this still an issue for anyone in this thread? Can't reproduce this with either the AppImage or Flatpak version of Ferdium.

It is for me. I'm on version 6.5.1 and the icon still doesn't appear, I installed it manually using the .deb.
I'm on Unity 7.5.1. though, not Gnome. I didn't check before, so that's probably why it stopped working since the time I reported. No idea how related that is, but likely is.

**Comment by thalesmaoa** [comment]
Same here! Phanteom and elementaryOS. Icon is missing.

**Comment by johncassil** [comment]
Still happening for me on Ubuntu 22.04.4 LTS
![image](https://github.com/ferdium/ferdium-app/assets/1057388/a1bcdbdb-e0ae-42aa-ac22-996fbe19fb3c)

**Comment by pH142857** [comment]
Same problem since latest update (Ferdium 7.2.1), GNOME 46, Wayland, Ubuntu 24.04 LTS.
Same problem with other Electron apps ([example](https://github.com/signalapp/Signal-Desktop/issues/7992)) so probably linked to an Electron bug (https://github.com/electron/electron/pull/52952).

**Comment by nenominal** [comment]
Can confirm, the issue persists with 7.2.2

**Comment by andrewtJMK** [comment]
Still persisting with 7.2.3 on Linux Mint 22.3 on Cinnamon 6.6.9 X11

**Comment by luxfoss** [comment]
I tested with the .deb and the appimage. Also with the flatpak package

.Deb and appimage worked, but the only one that didn't work was the flatpak, I personally think it could be an issue with flatpak sandboxs.

**Comment by RoestVrijStaal** [comment]
Yeah it's the "security" of Flatpak again to secure that the user's experience is horrible.

Related: https://github.com/flathub/org.ferdium.Ferdium/issues/79

I wish Flatpak had a "Panic"-button to just give the app all the permissions as native does when flatpak permissions go ill.
